deviations/forms.py

```diff
@@ -1,37 +1,77 @@
-from typing import Any
+from typing import Any, Dict
 
 from django import forms
 from django.utils.translation import gettext_lazy as _
 
 from aplus.api import api_reverse
 from exercise.models import BaseExercise
 from userprofile.models import UserProfile
-from course.models import CourseModule
+from course.models import CourseModule, UserTag
 from lib.fields import UsersSearchSelectField, SearchSelect
 from lib.widgets import DateTimeLocalInput
 
 
-class DeadlineRuleDeviationForm(forms.Form):
+class BaseDeviationForm(forms.Form):
+    """
+    Base class for deviation forms.
+    """
     module = forms.ModelMultipleChoiceField(
         queryset=CourseModule.objects.none(),
         required=False,
         label=_('LABEL_MODULE'),
-        help_text=_('DEVIATION_MODULE_HELPTEXT'),
         widget=SearchSelect,
     )
     exercise = forms.ModelMultipleChoiceField(
         queryset=BaseExercise.objects.none(),
         required=False,
         label=_('LABEL_EXERCISE'),
-        help_text=_('DEVIATION_EXERCISE_HELPTEXT'),
         widget=SearchSelect,
     )
+    submitter_tag = forms.ModelMultipleChoiceField(
+        queryset=UserTag.objects.none(),
+        required=False,
+        label=_('LABEL_SUBMITTER_TAG'),
+        widget=SearchSelect(display_fields=['slug']),
+    )
```

Comment:
The user tag field doesn't contain the hardcoded user tags ("Aalto" and "MOOC") because they aren't real tags that are stored in the database and thus not included in the queryset. Including them might be tricky.